jvm-runtime/ftl-runtime/pom.xml

```diff
@@ -293,5 +368,122 @@
+             deploying to Maven Central or other repositories that mandate signed artifacts for 
+             security and authenticity verification. Activate this profile when you need to create 
+             signed releases, usually in conjunction with the deployment to public repositories. -->
+        <profile>
+            <id>sign-artifacts</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.simplify4u.plugins</groupId>
+                        <artifactId>sign-maven-plugin</artifactId>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <!-- used to deploy artifacts to Maven Central via OSSRH (OSS Repository Hosting).
+             It configures the nexus-staging-maven-plugin for automated deployment and release,
+             and sets up the distribution management for both snapshot and release repositories.
+             Activate this profile when deploying open-source projects to Maven Central. -->
+        <profile>
+            <!-- https://central.sonatype.org/publish/publish-maven/#deployment -->
+            <id>ossrh</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.sonatype.plugins</groupId>
+                        <artifactId>nexus-staging-maven-plugin</artifactId>
+                        <version>1.6.13</version>
+                        <extensions>true</extensions>
+                        <configuration>
+                            <serverId>ossrh-releases</serverId>
+                            <nexusUrl>https://s01.oss.sonatype.org/</nexusUrl>
+                            <autoReleaseAfterClose>true</autoReleaseAfterClose>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+            <!-- specifies where the project artifacts should be distributed or deployed. -->
+            <!-- tells Maven where to deploy your project's artifacts when you run mvn deploy -->
+            <distributionManagement>
+                <snapshotRepository>
+                    <id>ossrh-snapshots</id>
+                    <url>https://s01.oss.sonatype.org/content/repositories/snapshots</url>
+                </snapshotRepository>
+                <!-- Sonatype's OSSRH - replicates to Maven Central within 30min of publish -->
+                <repository>
+                    <id>ossrh-releases</id>
+                    <url>https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+                </repository>
+            </distributionManagement>
+        </profile>
     </profiles>
+
+    <!-- specifies where the project artifacts should be distributed or deployed.
+         tells Maven where to deploy your project's artifacts when you run mvn deploy -->
+    <distributionManagement>
```

Comment:
Shouldn't this be in the .settings.xml? I also don't understand what the use case is here, won't the TBD artifactory pull them down from central once they have been released?